analyzer/optane/check_dimm_manageability

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+#set -x
+
+# Prevent repeated sourcing
+if [ -n "${__MODULE_CHECK_DIMM_ManageabilityState_}" ]; then
```

Comment:
All other modules use upper case for this, ie: `__MODULE_CHECK_DIMM_MANAGABILITYSTATE_`